src/components/common-components/folding-section/folding-section.component.tsx

```diff
@@ -11,15 +11,14 @@ interface FoldingSectionOptions {
 }
 
 export function FoldingSection({ name, children }: FoldingSectionOptions): JSX.Element {
-  const [opened, setOpened] = useState(true);
+  const [opened, setOpened] = useState(false);
   return (
     <div className="foldingSection">
       <div className="header">
         <div onClick={(): void => setOpened((prev) => !prev)} className="text">
           <span>{opened ? <ExpandMoreIcon /> : <ChevronRightIcon />}</span>
           <span>{name}</span>
         </div>
-        <hr className="middle" />
```

Comment:
na Figmie nie ma tej linii, więc ją usunęłam, ale wg mnie lepiej wyglądało z nią

Reply:
Zrób jak wygląda lepiej według Cb, ponieważ to programujesz i widzisz jak się komponuje.

Reply:
Zgadzam się z Pawłem